SingularityUI/app/components/machines/Slaves.jsx

```diff
@@ -344,6 +388,20 @@ const Slaves = (props) => {
 
   const inactiveSlaves = props.slaves.filter(({currentState}) => Utils.isIn(currentState.state, ['DEAD', 'MISSING_ON_STARTUP']));
 
+  const inactiveHostsPanel = (inactiveHosts) => (
+    inactiveHosts && inactiveHosts.length > 0 && (
+      <div className="row">
+        <h3>Inactive Hosts</h3>
+        <p>These hosts are marked as inactive: </p>
+        <ul className="list-group">
+          {inactiveHosts.map((host) => (
+            <li className="list-group-item" key={host}>{host}</li>
```

Comment:
might be good to be able to remove the host from this list in this location as well.